src/composables/Project.js

```diff
@@ -511,22 +511,26 @@ export async function deleteProjectFile(projectId, file, isFolder) {
 }
 
 /**
- * Get status of all files, except the unmodified files.
+ * Get the status of all files. If filePaths is defined, get the status of the files
+ * that strictly or partially match the given filePaths.
  * @param {String} projectId - Id of project.
+ * @param {String[]} filePaths - Limit the query to the given files and directories.
+ * @param {Function} filter - Filter to only return results whose filepath matches a given function.
  * @return {Promise<FileStatus[]>} All files status.
  */
-export async function getStatus(projectId) {
+export async function getStatus(projectId, filePaths, filter) {
   return git.statusMatrix({
     fs,
     dir: `/${projectId}`,
+    filepaths: filePaths,
```

Comment:
please rename `filePaths` to `filepaths` and make `filepaths,`

Reply:
done